app/src/main/java/com/example/ana/exampleapp/Variables.java

```diff
@@ -17,14 +17,13 @@
  */
 public final class Variables {
     // The server IP
-    private static final String IP = "IP";
+    private static final String IP = "localhost";
```

Comment:
In ok with this change as this will be most of the time the case, but then I will add in a comment called something like **Adapt the project to work in the development environment**